imap/jmap_contact.c

```diff
@@ -5349,6 +5403,30 @@ static void setaddressbook_readprops(jmap_req_t *req,
         jmap_parser_invalid(parser, "name");
     }
 
+    /* description */
+    jprop = json_object_get(arg, "description");
+    if (json_is_string(jprop)) {
+        props->description = json_string_value(jprop);
+    }
+    else if (JNOTNULL(jprop)) {
+        jmap_parser_invalid(parser, "description");
+    }
+
+    /* sortOrder */
+    jprop = json_object_get(arg, "sortOrder");
+    if (json_is_integer(jprop)) {
+        json_int_t t = json_integer_value(jprop);
+        if (t < 0 || t > INT_MAX) {
```

Comment:
Here you allow INT_MAX as a value but the fetch code requires the value be < INT_MAX.

Other existing code for a different type does:
```
imap/jmap_mailbox.c:        if (args->sortorder < 0 || args->sortorder >= INT_MAX) {

```
so probably here we should disallow INT_MAX and add a test to cass to make sure the expected value(s) at the border are handled right